api/models/site.js

```diff
@@ -175,6 +183,18 @@ module.exports = (sequelize, DataTypes) => {
       values: ['active', 'inactive'],
       defaultValue: 'active',
     },
+    cfInstanceName: {
```

Comment:
what is cfInstanceName?

Reply:
ok .. from below it appears to be a service instance? could "cfServiceName" be better?

Reply:
Sounds good

Reply:
or s3ServiceName .. i think is better

Reply:
Updated